int main()
{
    try{
        int a=0;
        int n;
        n=7/a;
    }
    catch("ArithmeticalExcption")
    {
        //n=0;
    }
}